drivers/i2c/Kconfig.tca9546a

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 Innoseis BV
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig I2C_TCA9546A
+	bool "I2C addressable switch"
+	help
+		Enable TCA9546A i2C bus switch
+
+if I2C_TCA9546A
+
+config  I2C_TCA9546_ROOT_INIT_PRIO
```

```suggestion
config I2C_TCA9546_ROOT_INIT_PRIO
```

Being strict about whitespace consistency in this case makes it possible to search for the definition of the config option using `git grep 'config I2C_TCA9546_ROOT_INIT_PRIO'` instead of messing around with variable whitespace.